src/methods/query/index.ts

```diff
@@ -109,13 +109,25 @@ const getAccountBalances = (account: any) => {
     _getAccountBalances(account)
       .then((balances: AssetBalance[]) => {
         resolve(
-          balances.map(({ assetId, balance, balanceFormatted }) => {
+          balances.map(({ assetId, balance, balanceFormatted, freeBalance, feeFrozenBalance, miscFrozenBalance, reservedBalance }) => {
             return {
               assetId,
               balance: toExternalBN(balance),
               balanceFormatted: toExternalBN(
                 new BigNumber(balanceFormatted)
               ).toString(),
+              freeBalance: toExternalBN(
+                new BigNumber(freeBalance)
+              ).toString(),
+              feeFrozenBalance: toExternalBN(
+                new BigNumber(feeFrozenBalance)
+              ).toString(),
+              miscFrozenBalance: toExternalBN(
+                new BigNumber(miscFrozenBalance)
+              ).toString(),
+              reservedBalance: toExternalBN(
+                new BigNumber(reservedBalance)
+              ).toString(),
```

Comment:
`freeBalance `, `feeFrozenBalance `, `miscFrozenBalance `, `reservedBalance ` - these balances should be in BigNumber format but not in string.